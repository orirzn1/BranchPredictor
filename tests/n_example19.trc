8 2 2 1 local_history local_tables using_share_lsb
0x4f330 T 0x548
0xc63c N 0xee8f8
0xc63c N 0xee8f8
0xc63c T 0xee8f8
0x4f330 N 0x548
0xc63c N 0xee8f8
0xc63c N 0xee8f8
0x4f330 N 0x548
0x4f330 N 0x548
0x4f330 N 0x548
0xc63c N 0xee8f8
0x4f330 N 0x548
0x4f330 N 0x548
0xc63c T 0xee8f8
0xc63c N 0xee8f8
0x4f330 N 0x548
0x4f330 N 0x548
0x4f330 N 0x548
0x4f330 T 0x548
0xc63c N 0xee8f8
0xc63c T 0xee8f8
0x4f330 T 0x548
0x4f330 T 0x548
0xc63c N 0xee8f8
0xc63c T 0xee8f8
0x4f330 T 0x548
0x4f330 T 0x548
0xc63c N 0xee8f8
0x4f330 T 0x548
0xc63c N 0xee8f8
0x4f330 N 0x548
0x4f330 T 0x548
0xc63c N 0xee8f8
0xc63c T 0xee8f8
0xc63c N 0xee8f8
0x4f330 T 0x548
0xc63c N 0xee8f8
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0x4f330 N 0x548
0xc63c N 0xee8f8
0x4f330 N 0x548
0x4f330 T 0x548
0xc63c N 0xee8f8
0x4f330 T 0x548
0x4f330 T 0x548
0xc63c T 0xee8f8
0x4f330 T 0x548
0x4f330 T 0x548
0xc63c N 0xee8f8
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0x4f330 T 0x548
0xc63c N 0xee8f8
0x4f330 T 0x548
0xc63c T 0xee8f8
0xc63c N 0xee8f8
0x4f330 N 0x548
0xc63c N 0xee8f8
0x4f330 T 0x548
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0x4f330 N 0x548
0x4f330 T 0x548
0xc63c T 0xee8f8
0x4f330 T 0x548
0xc63c N 0xee8f8
0xc63c T 0xee8f8
0xc63c N 0xee8f8
0x4f330 N 0x548
0xc63c N 0xee8f8
0xc63c T 0xee8f8
0xc63c N 0xee8f8
0xc63c N 0xee8f8
0xc63c N 0xee8f8
0x4f330 T 0x548
0xc63c T 0xee8f8
0x4f330 N 0x548
0xc63c N 0xee8f8
0xc63c N 0xee8f8
0x4f330 T 0x548
0xc63c N 0xee8f8
0xc63c T 0xee8f8
0xc63c N 0xee8f8
0x4f330 N 0x548
0x4f330 N 0x548
0xc63c N 0xee8f8
0x4f330 N 0x548
0x4f330 N 0x548
0x4f330 T 0x548
0x4f330 N 0x548
0x4f330 T 0x548
0x4f330 N 0x548
0x4f330 T 0x548
0xc63c T 0xee8f8
0x4f330 T 0x548
0x4f330 T 0x548
0xc63c T 0xee8f8
0x4f330 N 0x548
0xc63c N 0xee8f8
0xc63c T 0xee8f8
0x4f330 N 0x548
0x4f330 T 0x548
0x4f330 N 0x548
0x4f330 N 0x548
0x4f330 N 0x548
0x4f330 N 0x548
0x4f330 T 0x548
0xc63c N 0xee8f8
0x4f330 N 0x548
0xc63c N 0xee8f8
0x4f330 T 0x548
0x4f330 N 0x548
0xc63c T 0xee8f8
0xc63c N 0xee8f8
0x4f330 N 0x548
0x4f330 N 0x548
0xc63c N 0xee8f8
0x4f330 N 0x548
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0x4f330 T 0x548
0xc63c T 0xee8f8
0x4f330 T 0x548
0xc63c N 0xee8f8
0x4f330 N 0x548
0x4f330 T 0x548
0x4f330 T 0x548
0xc63c T 0xee8f8
0x4f330 T 0x548
0xc63c N 0xee8f8
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0xc63c N 0xee8f8
0x4f330 N 0x548
0xc63c T 0xee8f8
0x4f330 N 0x548
0xc63c T 0xee8f8
0xc63c N 0xee8f8
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0xc63c N 0xee8f8
0x4f330 N 0x548
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0x4f330 N 0x548
0xc63c T 0xee8f8
0x4f330 N 0x548
0x4f330 N 0x548
0x4f330 T 0x548
0xc63c N 0xee8f8
0xc63c N 0xee8f8
0xc63c T 0xee8f8
0x4f330 N 0x548
0x4f330 N 0x548
0x4f330 T 0x548
0xc63c T 0xee8f8
0x4f330 T 0x548
0x4f330 N 0x548
0x4f330 N 0x548
0xc63c T 0xee8f8
0xc63c N 0xee8f8
0xc63c N 0xee8f8
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0xc63c N 0xee8f8
0x4f330 N 0x548
0xc63c T 0xee8f8
0x4f330 N 0x548
0x4f330 T 0x548
0x4f330 N 0x548
0x4f330 T 0x548
0xc63c N 0xee8f8
0xc63c N 0xee8f8
0x4f330 N 0x548
0xc63c N 0xee8f8
0x4f330 N 0x548
0x4f330 T 0x548
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0x4f330 T 0x548
0x4f330 N 0x548
0xc63c N 0xee8f8
0xc63c N 0xee8f8
0x4f330 N 0x548
0x4f330 T 0x548
0x4f330 T 0x548
0x4f330 N 0x548
0x4f330 N 0x548
0xc63c N 0xee8f8
0x4f330 N 0x548
0x4f330 N 0x548
0xc63c N 0xee8f8
0x4f330 T 0x548
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0x4f330 N 0x548
0xc63c N 0xee8f8
0x4f330 N 0x548
0xc63c N 0xee8f8
0x4f330 T 0x548
0xc63c T 0xee8f8
0x4f330 T 0x548
0x4f330 N 0x548
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0x4f330 T 0x548
0xc63c N 0xee8f8
0x4f330 N 0x548
0xc63c T 0xee8f8
0xc63c N 0xee8f8
0x4f330 T 0x548
0xc63c N 0xee8f8
0xc63c T 0xee8f8
0x4f330 T 0x548
0xc63c N 0xee8f8
0xc63c N 0xee8f8
0x4f330 T 0x548
0x4f330 T 0x548
0x4f330 T 0x548
0x4f330 N 0x548
0x4f330 N 0x548
0xc63c N 0xee8f8
0xc63c N 0xee8f8
0xc63c N 0xee8f8
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0x4f330 T 0x548
0x4f330 T 0x548
0xc63c N 0xee8f8
0x4f330 T 0x548
0xc63c T 0xee8f8
0xc63c N 0xee8f8
0x4f330 N 0x548
0x4f330 N 0x548
0xc63c N 0xee8f8
0x4f330 T 0x548
0x4f330 T 0x548
0x4f330 N 0x548
0xc63c N 0xee8f8
0x4f330 N 0x548
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0x4f330 T 0x548
0x4f330 N 0x548
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0x4f330 N 0x548
0x4f330 N 0x548
0x4f330 N 0x548
0xc63c T 0xee8f8
0xc63c N 0xee8f8
0xc63c N 0xee8f8
0x4f330 N 0x548
0xc63c T 0xee8f8
0x4f330 T 0x548
0x4f330 T 0x548
0x4f330 T 0x548
0xc63c T 0xee8f8
0xc63c N 0xee8f8
0xc63c N 0xee8f8
0xc63c T 0xee8f8
0x4f330 N 0x548
0xc63c T 0xee8f8
0x4f330 N 0x548
0x4f330 T 0x548
0xc63c N 0xee8f8
0xc63c T 0xee8f8
0x4f330 T 0x548
0x4f330 N 0x548
0x4f330 N 0x548
0xc63c T 0xee8f8
0x4f330 N 0x548
0x4f330 T 0x548
0x4f330 T 0x548
0x4f330 T 0x548
0xc63c T 0xee8f8
0xc63c N 0xee8f8
0xc63c N 0xee8f8
0x4f330 T 0x548
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0x4f330 T 0x548
0x4f330 T 0x548
0x4f330 N 0x548
0x4f330 N 0x548
0xc63c T 0xee8f8
0xc63c N 0xee8f8
0xc63c N 0xee8f8
0x4f330 N 0x548
0xc63c T 0xee8f8
0x4f330 T 0x548
0x4f330 T 0x548
0x4f330 N 0x548
0xc63c T 0xee8f8
0x4f330 T 0x548
0x4f330 N 0x548
0x4f330 T 0x548
0xc63c N 0xee8f8
0x4f330 T 0x548
0x4f330 T 0x548
0xc63c T 0xee8f8
0x4f330 T 0x548
0x4f330 T 0x548
0xc63c N 0xee8f8
0xc63c T 0xee8f8