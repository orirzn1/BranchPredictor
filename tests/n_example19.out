0x4f330 N 0x4f334
0xc63c N 0xc640
0xc63c N 0xc640
0xc63c N 0xc640
0x4f330 N 0x4f334
0xc63c N 0xc640
0xc63c N 0xc640
0x4f330 N 0x4f334
0x4f330 T 0x548
0x4f330 N 0x4f334
0xc63c N 0xc640
0x4f330 N 0x4f334
0x4f330 N 0x4f334
0xc63c N 0xc640
0xc63c N 0xc640
0x4f330 N 0x4f334
0x4f330 N 0x4f334
0x4f330 N 0x4f334
0x4f330 N 0x4f334
0xc63c N 0xc640
0xc63c N 0xc640
0x4f330 N 0x4f334
0x4f330 N 0x4f334
0xc63c N 0xc640
0xc63c N 0xc640
0x4f330 T 0x548
0x4f330 T 0x548
0xc63c N 0xc640
0x4f330 T 0x548
0xc63c N 0xc640
0x4f330 T 0x548
0x4f330 N 0x4f334
0xc63c T 0xee8f8
0xc63c N 0xc640
0xc63c N 0xc640
0x4f330 N 0x4f334
0xc63c N 0xc640
0xc63c T 0xee8f8
0xc63c N 0xc640
0x4f330 T 0x548
0xc63c N 0xc640
0x4f330 N 0x4f334
0x4f330 N 0x4f334
0xc63c N 0xc640
0x4f330 T 0x548
0x4f330 N 0x4f334
0xc63c T 0xee8f8
0x4f330 T 0x548
0x4f330 T 0x548
0xc63c N 0xc640
0xc63c N 0xc640
0xc63c N 0xc640
0x4f330 T 0x548
0xc63c N 0xc640
0x4f330 T 0x548
0xc63c N 0xc640
0xc63c N 0xc640
0x4f330 T 0x548
0xc63c T 0xee8f8
0x4f330 N 0x4f334
0xc63c T 0xee8f8
0xc63c N 0xc640
0xc63c N 0xc640
0xc63c N 0xc640
0x4f330 T 0x548
0x4f330 N 0x4f334
0xc63c T 0xee8f8
0x4f330 T 0x548
0xc63c T 0xee8f8
0xc63c N 0xc640
0xc63c N 0xc640
0x4f330 T 0x548
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0xc63c N 0xc640
0xc63c N 0xc640
0xc63c T 0xee8f8
0x4f330 T 0x548
0xc63c T 0xee8f8
0x4f330 T 0x548
0xc63c N 0xc640
0xc63c N 0xc640
0x4f330 T 0x548
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0xc63c N 0xc640
0x4f330 T 0x548
0x4f330 T 0x548
0xc63c N 0xc640
0x4f330 T 0x548
0x4f330 N 0x4f334
0x4f330 N 0x4f334
0x4f330 N 0x4f334
0x4f330 T 0x548
0x4f330 N 0x4f334
0x4f330 T 0x548
0xc63c T 0xee8f8
0x4f330 N 0x4f334
0x4f330 N 0x4f334
0xc63c N 0xc640
0x4f330 T 0x548
0xc63c T 0xee8f8
0xc63c N 0xc640
0x4f330 T 0x548
0x4f330 N 0x4f334
0x4f330 N 0x4f334
0x4f330 T 0x548
0x4f330 T 0x548
0x4f330 N 0x4f334
0x4f330 N 0x4f334
0xc63c N 0xc640
0x4f330 N 0x4f334
0xc63c N 0xc640
0x4f330 N 0x4f334
0x4f330 N 0x4f334
0xc63c T 0xee8f8
0xc63c N 0xc640
0x4f330 T 0x548
0x4f330 N 0x4f334
0xc63c N 0xc640
0x4f330 N 0x4f334
0xc63c T 0xee8f8
0xc63c N 0xc640
0xc63c N 0xc640
0x4f330 N 0x4f334
0xc63c T 0xee8f8
0x4f330 N 0x4f334
0xc63c T 0xee8f8
0x4f330 N 0x4f334
0x4f330 N 0x4f334
0x4f330 N 0x4f334
0xc63c N 0xc640
0x4f330 N 0x4f334
0xc63c N 0xc640
0xc63c N 0xc640
0xc63c N 0xc640
0xc63c T 0xee8f8
0x4f330 N 0x4f334
0xc63c T 0xee8f8
0x4f330 T 0x548
0xc63c N 0xc640
0xc63c N 0xc640
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0xc63c N 0xc640
0xc63c N 0xc640
0x4f330 N 0x4f334
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0x4f330 N 0x4f334
0xc63c N 0xc640
0x4f330 N 0x4f334
0x4f330 N 0x4f334
0x4f330 N 0x4f334
0xc63c N 0xc640
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0x4f330 T 0x548
0x4f330 N 0x4f334
0x4f330 N 0x4f334
0xc63c T 0xee8f8
0x4f330 N 0x4f334
0x4f330 N 0x4f334
0x4f330 N 0x4f334
0xc63c N 0xc640
0xc63c N 0xc640
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0xc63c N 0xc640
0x4f330 T 0x548
0xc63c N 0xc640
0x4f330 N 0x4f334
0x4f330 N 0x4f334
0x4f330 T 0x548
0x4f330 N 0x4f334
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0x4f330 N 0x4f334
0xc63c T 0xee8f8
0x4f330 N 0x4f334
0x4f330 N 0x4f334
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0xc63c N 0xc640
0xc63c N 0xc640
0x4f330 N 0x4f334
0x4f330 N 0x4f334
0xc63c T 0xee8f8
0xc63c N 0xc640
0x4f330 N 0x4f334
0x4f330 T 0x548
0x4f330 N 0x4f334
0x4f330 N 0x4f334
0x4f330 N 0x4f334
0xc63c T 0xee8f8
0x4f330 T 0x548
0x4f330 T 0x548
0xc63c T 0xee8f8
0x4f330 N 0x4f334
0xc63c N 0xc640
0xc63c T 0xee8f8
0x4f330 T 0x548
0xc63c N 0xc640
0x4f330 N 0x4f334
0xc63c N 0xc640
0x4f330 T 0x548
0xc63c T 0xee8f8
0x4f330 N 0x4f334
0x4f330 N 0x4f334
0xc63c T 0xee8f8
0xc63c N 0xc640
0x4f330 N 0x4f334
0xc63c N 0xc640
0x4f330 T 0x548
0xc63c N 0xc640
0xc63c T 0xee8f8
0x4f330 N 0x4f334
0xc63c N 0xc640
0xc63c T 0xee8f8
0x4f330 N 0x4f334
0xc63c T 0xee8f8
0xc63c N 0xc640
0x4f330 N 0x4f334
0x4f330 N 0x4f334
0x4f330 T 0x548
0x4f330 T 0x548
0x4f330 T 0x548
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0xc63c N 0xc640
0xc63c N 0xc640
0xc63c N 0xc640
0x4f330 T 0x548
0x4f330 T 0x548
0xc63c N 0xc640
0x4f330 T 0x548
0xc63c N 0xc640
0xc63c T 0xee8f8
0x4f330 T 0x548
0x4f330 N 0x4f334
0xc63c N 0xc640
0x4f330 T 0x548
0x4f330 T 0x548
0x4f330 T 0x548
0xc63c N 0xc640
0x4f330 N 0x4f334
0xc63c N 0xc640
0xc63c N 0xc640
0x4f330 T 0x548
0x4f330 T 0x548
0xc63c N 0xc640
0xc63c N 0xc640
0x4f330 N 0x4f334
0x4f330 T 0x548
0x4f330 T 0x548
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0xc63c N 0xc640
0x4f330 N 0x4f334
0xc63c N 0xc640
0x4f330 N 0x4f334
0x4f330 T 0x548
0x4f330 N 0x4f334
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0xc63c N 0xc640
0xc63c T 0xee8f8
0x4f330 T 0x548
0xc63c T 0xee8f8
0x4f330 N 0x4f334
0x4f330 N 0x4f334
0xc63c N 0xc640
0xc63c N 0xc640
0x4f330 T 0x548
0x4f330 N 0x4f334
0x4f330 N 0x4f334
0xc63c T 0xee8f8
0x4f330 T 0x548
0x4f330 N 0x4f334
0x4f330 T 0x548
0x4f330 N 0x4f334
0xc63c N 0xc640
0xc63c N 0xc640
0xc63c N 0xc640
0x4f330 N 0x4f334
0xc63c T 0xee8f8
0xc63c T 0xee8f8
0x4f330 T 0x548
0x4f330 T 0x548
0x4f330 T 0x548
0x4f330 N 0x4f334
0xc63c N 0xc640
0xc63c N 0xc640
0xc63c N 0xc640
0x4f330 T 0x548
0xc63c T 0xee8f8
0x4f330 N 0x4f334
0x4f330 T 0x548
0x4f330 T 0x548
0xc63c T 0xee8f8
0x4f330 N 0x4f334
0x4f330 T 0x548
0x4f330 N 0x4f334
0xc63c N 0xc640
0x4f330 T 0x548
0x4f330 N 0x4f334
0xc63c N 0xc640
0x4f330 T 0x548
0x4f330 T 0x548
0xc63c T 0xee8f8
0xc63c N 0xc640
flush_num: 150, br_num: 312, size: 344b
